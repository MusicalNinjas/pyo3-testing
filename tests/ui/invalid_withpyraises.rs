use pyo3_testing::with_py_raises;

fn test_raises() {
    with_py_raises!(PyTypeError { () });
}

fn main() {}
