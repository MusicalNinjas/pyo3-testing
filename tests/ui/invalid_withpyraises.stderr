error: Expected a comma (`,`) after this:
       Correct format for with_py_raises is: `Error Type` `Comma: [,]` `{block in braces}`
       E.g.: `with_py_raises!(PyTypeError, { addone.call1(("4",)) })`
 --> tests/ui/invalid_withpyraises.rs:4:21
  |
4 |     with_py_raises!(PyTypeError { () });
  |                     ^^^^^^^^^^^
